### ISSUE TITLE
[Feature Request] Notify multiple users for same item

### ISSUE THREAD

**Issue by qwasy** [Fallenbagel/jellyseerr#375]
### Description

If an item is requested by user1, user2 can't request it and will get no notification when it's available. 
Adding a button like "notify me aswell" would be nice for user2. 

### Desired Behavior

user1 and user2 get a notification when the item is available.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
sct/overseerr#789

**Comment by alvitali** [comment]
hey @fallenbagel 

i feel like the `upstream pending` label is somewhat inappropriate for this issue, as the maintainers of overseerr stated on multiple occasions they consider the project to be "feature complete" and development, especially regarding new features has slowed to a crawl, making this unlikely to happen in the near future.

i'd love to contribute myself to make this request happen, but i don't have the skills yet to tackle something as complex as this on an existing codebase.

i'd like to appeal for a reconsideration of this feature request, seeing as you introduced new features like the watchlist, which haven't made it into the upstream project.
while most jellyseerr admins might not profit from this feature (having access to all the data / notification endpoints anyways) it would be a pretty substantial QOL improvement for many users, especially on instances with a larger userbase

**Comment by fallenbagel** [comment]
> hey @fallenbagel 
> 
> i feel like the `upstream pending` label is somewhat inappropriate for this issue, as the maintainers of overseerr stated on multiple occasions they consider the project to be "feature complete" and development, especially regarding new features has slowed to a crawl, making this unlikely to happen in the near future.
> 

That's not true. We are in communication (and are pretty close) with overseerr, and and they've resumed development.

As for why this had upstream pending label, that was a long time ago. We used to be feature parity. Nothing is stopping anyone from opening new prs for new features (as you can see from our prs). 

Anyone can open a pr for a new feature from feature request list or something new ;)

**Comment by alvitali** [comment]
that's great to know, thanks! i might have to take a crack at it myself some day then ;)

**Comment by anhaehne** [comment]
I would like this feature as well so i started with a quick prototype.
I think the easiest solution would be to allow duplicate requests. This way multile users can request the same item.

I'm not using sonarr or radarr so i don't know how this would affect the integrations. Otherwise this works well for my usecase.

**Comment by Piddox** [comment]
@fallenbagel I have a question about a solution your wrote in https://github.com/fallenbagel/jellyseerr/pull/1673#issuecomment-2927011437:

> > The last case can be avoided when User B created their own Request instead of subscribing
> 
> My plans were if User A deletes the request, the subscription changes into a request. To avoid multiple requests as that would require a substantial amount of refactor of the core logic to not break *arr.
> 
> A subscription based model makes more sense imo

Let's say we're In the scenario where you have a separate radarr instance for HD and 4K.
And let's say users A and B users have requested a movie, user A requested it in HD format and user B in 4K format. Now user C comes along and hits the subscribe button. User A cancels his request, so only the 4K is still requested. What happens to the subscription of user C? Does it become a request for the HD version? Or does it remain a subscription for the 4K?

What I think there should be is two subscription buttons in this case: one for the HD version (if it is already requested) and for the 4K version (if it is already requested). Then the user can decide to subscribe to both, and your solution of switching the subscription to a request in case the request gets deleted still works. But let me know what you think as well.

**Comment by herkalurk** [comment]
Any movement on this would be good.  This was behavior my users had with OMBI where they would also be notified as soon as a request was completed along with original requestor.  Would like to see something similar in seerr.

**Comment by Crosus97** [comment]
I was almost to request this same Featured

There is any upcoming update about this?

**Comment by Montago** [comment]
Any news on this ? .. it's been 3 years since original feature request...

**Comment by InfAmoUsaeble** [comment]
Also hoping this could be implemented!

Thanks for all your work btw
